Session/Conversations/Settings/ThreadSettingsViewModel.swift

```diff
@@ -233,192 +308,149 @@ class ThreadSettingsViewModel: SessionTableViewModel, NavigationItemSource, Navi
+                                                    ),
+                                                    generator: { SessionProBadge(size: .mini) }
+                                                )
+                                            )
+                                        default:
+                                            return .generic(renew: dependencies[singleton: .sessionProState].isSessionProExpired)
+                                    }
+                                }()
+                                
+                                dependencies[singleton: .sessionProState].showSessionProCTAIfNeeded(
+                                    proCTAModalVariant,
+                                    onConfirm: {
+                                        dependencies[singleton: .sessionProState].showSessionProBottomSheetIfNeeded(
+                                            presenting: { bottomSheet in
+                                                viewModel.transitionToScreen(bottomSheet, transitionType: .present)
+                                            }
+                                        )
+                                    },
+                                    presenting: { modal in
+                                        viewModel.transitionToScreen(modal, transitionType: .present)
                                     }
                                 )
-                            },
-                            presenting: { modal in
-                                self?.transitionToScreen(modal, transitionType: .present)
                             }
                         )
-                    }
+                    ),
+                    accessibility: Accessibility(
+                        identifier: "Username",
+                        label: threadViewModel.displayName
+                    )
                 ),
                 
                 (threadViewModel.displayName == threadViewModel.contactDisplayName ? nil :
-                    SessionCell.Info(
+                    SessionListScreenContent.ListItemInfo(
                         id: .contactName,
-                        subtitle: SessionCell.TextInfo(
-                            "(\(threadViewModel.contactDisplayName))", // stringlint:ignore
-                            font: .subtitle,
-                            alignment: .center
-                        ),
-                        styling: SessionCell.StyleInfo(
-                            tintColor: .textSecondary,
-                            customPadding: SessionCell.Padding(
-                                top: 0,
-                                bottom: Values.largeSpacing
-                            ),
-                            backgroundStyle: .noBackground
+                        variant: .cell(
+                            info: .init(
+                                title: .init(
```

Comment:
So this is a really minor thing but using `.init(` instead of `SessionListScreenContent.TextInfo` makes it a little harder to understand the code quickly, especially if there are multiple `.init(` calls in the same chunk of code like this - mostly because I can't tell at a glance _what_ is being initialised here, if this was a generic function it would be even harder to tell (though I'm not sure if Swift would allow `.init(` with generics 🤔)